group.go

```diff
@@ -40,22 +39,16 @@ import (
 // name in HelpText strings.
 const BinaryName = "{{.Name}}"
 
-// Error allows for creating constant errors instead of sentinel ones.
-type Error string
-
-// Error implements error.
-func (e Error) Error() string { return string(e) }
-
 // ErrBailEarlyRequest is returned when a call to RunConfig was successful but
 // signals that the application should exit in success immediately.
 // It is typically returned on --version and --help requests that have been
 // served. It can and should be used for custom config phase situations where
 // the job of the application is done.
-const ErrBailEarlyRequest Error = "exit request from flag handler"
+var ErrBailEarlyRequest = errors.New("exit request from flag handler")
```

Comment:
This change means people can override the error or even nil them, as the error variables are exported.